pygmt/datasets/load_remote_dataset.py

```diff
@@ -3,32 +3,35 @@
 """
 from __future__ import annotations
 
-from typing import NamedTuple
+from typing import TYPE_CHECKING, ClassVar, NamedTuple
 
 from pygmt.exceptions import GMTInvalidInput
 from pygmt.helpers import kwargs_to_strings
 from pygmt.io import load_dataarray
 from pygmt.src import grdcut, which
 
+if TYPE_CHECKING:
+    import xarray as xr
+
 
 class Resolution(NamedTuple):
     """
-    The available grid registrations for a given resolution and whether it is a tiled
-    grid.
+    Resolution code, the available grid registrations and whether it is tiled.
 
     Attributes
     ----------
+    code : str
+        The resolution code.
```

Comment:
Resolution 'code' sounds a bit weird to me, but ok since it is used at https://www.generic-mapping-tools.org/remote-datasets/earth-gebco.html#usage. Just add some examples to make it clear.

```suggestion
        The resolution code. E.g. '01d', '30m', '01s'.
```